geotrek/cirkwi/parsers.py

```diff
@@ -0,0 +1,302 @@
+import xml.etree.ElementTree as ET
+
+from django.conf import settings
+from django.contrib.gis.geos import Point, MultiPoint, GEOSGeometry
+from django.utils.translation import gettext as _
+
+from geotrek.common.utils.parsers import get_geom_from_gpx
+from geotrek.trekking.models import DifficultyLevel
+from geotrek.cirkwi.models import CirkwiLocomotion
+from geotrek.common.parsers import AttachmentParserMixin, Parser, RowImportError
+from geotrek.tourism.models import TouristicContent, TouristicContentType1
+from geotrek.trekking.models import Trek, Practice
+
+
+class CirkwiParser(AttachmentParserMixin, Parser):
+    """
```

Comment:
I suggest to expand the docstring with explanations regarding the new XML-related possibilities for the fields mapping. Here is a proposal :

```
The CirkwiParser class handles some XML-related specificities for the fields mapping:

- `src` values should follow an XML path syntax so the separator is '/' instead of '.'
- `src` value "locomotions/locomotion" gets the text value of the first <locomotion> XML element encountered (inside a <locomotions> element at the row's root level)
- "informations/information[@langue='fr']/titre" means we get the text value of the first <titre> element having a parent <information> with attribute `langue` equals "fr"
- the "<LANG>" marker is replace with the `default_language` of the parser
- a '@@' sequence at the end followed by a attribute name indicates the fetched value should be the value of the attribute (for instance "locomotions/locomotion@@type" will get the value of the 'type' attribute of the <locomotion> element)
- finally the '/*' sequence at the end of a `src` value indicates a list of all XML elements matching the path should be returned (so "locomotions/locomotion/*" has the same meaning than "locomotions.*.locomotion" from the base Parser class)

See the overridden `CirkwiParser.get_part` method for the implementation.
```

Probably too verbose feel free to include it or not as you like.